src/devices/cpu/m6502/n2a03.h

```diff
@@ -15,21 +15,16 @@
 #include "m6502.h"
 #include "sound/nes_apu.h"
 
-class n2a03_device : public m6502_device, public device_mixer_interface {
+class n2a03_base_device : public m6502_device {
```

Comment:
It might be better to call this `n2a03_core_device` or `n2a03_cpu_device` something to avoid someone potentially interpreting `_base` in the name as an abstract base class for devices.